tox.ini

```diff
@@ -58,6 +58,6 @@ deps=
 deps=
     {[testenv]deps}
     Django>=1.8,<1.9
-    djangorestframework>=3.3.2
+    djangorestframework>=3.3.2,<3.7
```

Comment:
so using < 3.7 helps to pass more tests?

Reply:
@auvipy yes because django rest framework 3.7 dropped support of django 1.8. See <https://github.com/encode/django-rest-framework/issues/5491>.

Reply:
OK great. I will also drop less then 1.11 when 2.0 support would be fully added